src/base/listener.ts

```diff
@@ -411,7 +411,7 @@ class Listener extends EventEmitter implements InterfaceListener {
     try {
       maConn = TCPConnection.fromSocket(socket, this.peerId) as any
     } catch (err) {
-      error(`inbound connection failed. ${err.message}`)
+      error(`inbound connection failed. ${err instanceof Error ? err.message : err}`)
```

Comment:
This seems as a copy-paste anti-pattern. Is there a better way to serialize errors ?